roadmap/implementors-guide/src/node/availability/bitfield-signing.md

```diff
@@ -19,7 +23,8 @@ Localized to a specific relay-parent `r`
 If not running as a validator, do nothing.
 
 - Determine our validator index `i`, the set of backed candidates pending availability in `r`, and which bit of the bitfield each corresponds to.
-- > TODO: wait T time for availability distribution?
 - Start with an empty bitfield. For each bit in the bitfield, if there is a candidate pending availability, query the [Availability Store](../utility/availability-store.md) for whether we have the availability chunk for our validator index.
 - For all chunks we have, set the corresponding bit in the bitfield.
 - Sign the bitfield and dispatch a `BitfieldDistribution::DistributeBitfield` message.
+
+Note that because the bitfield signing job is launched once per block and executes immediately, the signed availability statement for block `N` is best considered as a report of the candidates available at the end of block `N-1`.
```

Comment:
https://github.com/paritytech/polkadot/pull/1320#discussion_r448562641 is trying to address this actually. For throughput, it might be better to wait a bit before signing / distributing bitfields so that some availability work can happen before the relay-chain block is produced. We have to wait a little bit for candidate backing in the `Proposer` anyway.

Reply:
https://github.com/paritytech/polkadot/pull/1320#issuecomment-654145821 Jeff left a comment earlier detailing his response

Reply:
> Yes, we'd want nodes to delay issuing their first bitfield on some relay chain block until they can recieve some pieces, but neither should they get stuck between this delay, advancing the relay chain block on which they build bitfields, and the torrent of new blocks.

Yup, each relay-chain block is going to be kicking off a job for signing a new bitfield, so we won't get stuck in the delay.

As a "SWAG" I'd say it's reasonable to wait 1500ms before issuing bitfields, but I also have absolutely no basis for this.